README.rst

```diff
@@ -72,6 +72,8 @@ This is the list of checks that are currently performed:
 
 * You should have classifiers specifying the supported Python versions.
 
+* You should have 'python_requires' specifying the supported Python versions.
```

Comment:
Instead of only using a legacy tool-specific key used only by Setuptools (which itself is ditching it eventually now that it has moved to PEP 621 itself), you should at the very least mention the standard PEP 621 key here. Also, use proper reST syntax.

```suggestion
* You should have ``requires-python``/``python_requires``
  specifying the Python versions you support.
```